util/cache.c

```diff
@@ -3,6 +3,10 @@
 #include <sddf/util/cache.h>
 #include <sddf/util/util.h>
 
+#ifndef CONFIG_AARCH64_USER_CACHE_ENABLE
+#error "CONFIG_AARCH64_USER_CACHE_ENABLE must be enabled"
```

Comment:
```suggestion
#error "CONFIG_AARCH64_USER_CACHE_ENABLE must be enabled"
#error "seL4 must be configured with CONFIG_AARCH64_USER_CACHE_ENABLE"
```

It's a kind of random config option so just want to make sure it's clear to people that it's an seL4 config option.